CHANGELOG.md

```diff
@@ -2,6 +2,100 @@
+- **Documentation:** Import code and commit history from the docs repository [\#5544](https://github.com/consuldemocracy/consuldemocracy/pull/5544)
+- **Homepage:** Allow sorting homepage cards [\#5276](https://github.com/consuldemocracy/consuldemocracy/pull/5276)
+- **Maintenance:** Use Zeitwerk to autoload files [\#5425](https://github.com/consuldemocracy/consuldemocracy/pull/5425) and [\#5492](https://github.com/consuldemocracy/consuldemocracy/pull/5492)
+- **Maintenance-CI:** Run MDL in GitHub actions [\#5582](https://github.com/consuldemocracy/consuldemocracy/pull/5582)
+- **Maintenance-CI:** Run every linter separately in Github Actions [\#5545](https://github.com/consuldemocracy/consuldemocracy/pull/5545) and [\#5583](https://github.com/consuldemocracy/consuldemocracy/pull/5583)
+- **Maintenance-Deployment:** Clear Rails cache when upgrading Consul Democracy [\#5547](https://github.com/consuldemocracy/consuldemocracy/pull/5547)
+- **Maps:** Geozone Admin maps [\#5390](https://github.com/consuldemocracy/consuldemocracy/pull/5390)
+- **Multitenancy:** Allow different locales per tenant [\#5243](https://github.com/consuldemocracy/consuldemocracy/pull/5243) and [\#5488](https://github.com/consuldemocracy/consuldemocracy/pull/5488)
+
+### Changed
+
+- **Accessibility:** Replace initialjs-rails with custom avatar code [\#5481](https://github.com/consuldemocracy/consuldemocracy/pull/5481)
+- **Accessibility:** Use buttons to open/close admin navigation submenus [\#5460](https://github.com/consuldemocracy/consuldemocracy/pull/5460)
+- **Accessibility:** Use buttons for non-GET actions in the admin section [\#5459](https://github.com/consuldemocracy/consuldemocracy/pull/5459)
+- **Admin:** Adjust permissions in admin locales [\#5585](https://github.com/consuldemocracy/consuldemocracy/pull/5585)
+- **Maintenance:** Base staging configuration on the production one [\#5463](https://github.com/consuldemocracy/consuldemocracy/pull/5463)
+- **Maintenance:** Update code of conduct contact address [\#5442](https://github.com/consuldemocracy/consuldemocracy/pull/5442)
+- **Maintenance-CI:** Replace SCSS Lint with Stylelint [\#5448](https://github.com/consuldemocracy/consuldemocracy/pull/5448)
+- **Maintenance-Deployment:** \[Security\] Upgrade Ruby to version 3.2.4 [\#5541](https://github.com/consuldemocracy/consuldemocracy/pull/5541)
+- **Maintenance-Deployment:** Don't ping search engines after sitemap generation [\#5464](https://github.com/consuldemocracy/consuldemocracy/pull/5464)
+- **Maintenance-Deployment:** Upgrade Ruby to version 3.2.3 [\#5404](https://github.com/consuldemocracy/consuldemocracy/pull/5404)
+- **Maintenance-Gems:** \[Security\] Bump rails from 7.0.8.1 to 7.0.8.4 [\#5570](https://github.com/consuldemocracy/consuldemocracy/pull/5570)
+- **Maintenance-Gems:** Bump rexml from 3.2.6 to 3.2.8 [\#5542](https://github.com/consuldemocracy/consuldemocracy/pull/5542)
```

Comment:
I forgot to add a "Security" label to this pull request (I've just added it) :pray:.